src/main/java/org/opensearch/flowframework/workflow/RetryableWorkflowStep.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.workflow;
+
+import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.common.settings.Settings;
+
+import static org.opensearch.flowframework.common.FlowFrameworkSettings.MAX_REQUEST_RETRY;
+
+/**
+ * Abstract retryable workflow step
+ */
+public abstract class RetryableWorkflowStep implements WorkflowStep {
```

Comment:
AbstractClass should begin with the keyword `Abstract`
```suggestion
public abstract class AbstractRetryableWorkflowStep implements WorkflowStep {
```